phoebe/frontend/bundle.py

```diff
@@ -13057,7 +13057,7 @@ def _write_export_solver_script(self, script_fname, out_fname, solver, solution,
         exclude_solvers = [s for s in self.solvers if s!=solver]
         solver_ps = self.get_solver(solver=solver, **_skip_filter_checks)
         if 'compute' in solver_ps.qualifiers:
-            compute = solver_ps.get_value(qualifier='compute', compute=kwargs.get('compute', None), default=[], **_skip_filter_checks)
+            compute = kwargs.get('compute', solver_ps.get_value(qualifier='compute', **_skip_filter_checks))
```

Comment:
To override a parameter value from a kwarg, we usually call `parameter_name=kwargs.get('parameter_name', None)` which results in returning the value in the kwarg if provided, and otherwise the parameter value itself.  However, in this case, the parameter name is `compute` but that clashes with the compute _tag_ resulting in no results.  The new line should result in the same intended logic, but bypasses the filter entirely and prevents the tag collision. 

The only downside to this approach being used widely is that it also bypasses validity checks, etc, but in this case those would result in errors in the following line.  If this ever became a problem, we could add logic to ensure that the user-passed value for compute is a valid option and raise a useful error if it is not.